code/game/objects/structures/extinguisher.dm

```diff
@@ -165,9 +165,9 @@
 	else
 		icon_state = "extinguisher_empty"
 
-/obj/structure/extinguisher_cabinet/empty/New(turf/loc, ndir = null)
+/obj/structure/extinguisher_cabinet/empty/New(turf/loc, direction = null)
 	extinguishertype = NO_EXTINGUISHER
```

Comment:
Know per chance why this is put in here and not overriden in the object define?

Reply:
Actually its because this is some weird hybrid subtype define and `New()` proc merged into one

Reply:
Oh like no subtype definition? You can just make one then and remove this New()